libs/ledger-live-common/src/families/stacks/setup.ts

```diff
@@ -2,7 +2,7 @@
 
 import { createBridges } from "@ledgerhq/coin-stacks";
 import stacksResolver, { signMessage } from "@ledgerhq/coin-stacks/signer/index";
-import makeCliTools from "@ledgerhq/coin-xrp/test/cli";
+import makeCliTools from "@ledgerhq/coin-stacks/test/cli";
```

Comment:
Is it intentional?

Reply:
No it was a mix up

Reply:
yes that was the point of this PR at first then @Wozacosta fixed playwright